tnth/prodanon/prodanon/config.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+class Config(object):
+    db_host = os.environ.get("POSTGRES_HOST", '127.0.0.1')
+    db_port = os.environ.get("POSTGRES_PORT", '5432')
+    db_name = os.environ.get("POSTGRES_DB")
```

Comment:
Does this need to be manually configured? Copied an existing default just in case
```suggestion
    db_name = os.environ.get("POSTGRES_DB", "portaldb")
```